src/Oro/Bundle/CacheBundle/Resources/config/services.yml

```diff
@@ -5,6 +5,13 @@ services:
     oro_cache.oro_data_cache_manager:
         class: %oro_cache.oro_data_cache_manager.class%
 
+    oro_cache.config_cache_warmer.listener:
+        class: 'Oro\Bundle\CacheBundle\EventListener\CacheWarmerListener'
+        arguments:
+            - '%kernel.debug%'
+        tags:
+          - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
```

Comment:
It should also work for console commands, for example action groups can be used in workflow transitions which can be used in a cli process